package.json

```diff
@@ -85,7 +85,7 @@
     "istanbul-lib-coverage": "3.2.0",
     "jest-docblock": "^29.7.0",
     "koalas": "^1.0.2",
-    "limiter": "^1.1.4",
+    "limiter": "1.1.5",
```

Comment:
Most other dependencies are not fixed. Why should this one be different? If we want to use ranges we should continue to use ranges, and if we don't want ranges we might consider fixing all versions.

Reply:
FTR: I'll propose pinning all versions later on yes :+1: